### PR TITLE
[AS-5461]added support for new plugin

### PR DESCRIPTION
**Pull request by bhanuc**
https://aftership.atlassian.net/browse/AS-5461

### CODE REVIEW

**Comment by Bossa573** [comment]
the travis check fail, should be normal right?
coz there's no even .travis.yml

**Comment by bhanuc** [comment]
Yes. No tests exits for this repo it seems.

File: api/class-aftership-api-orders.php [hunk 1]
```diff
@@ -265,10 +265,21 @@ public function get_order($id, $fields = null)
...
                 'tracking_destination_country' => get_post_meta($order->id, '_aftership_tracking_destination_country', true),
 			);
-		} else { //$49
-			$order_data['aftership']['woocommerce']['trackings'][] = array(
-				'tracking_number' => get_post_meta($order->id, '_tracking_number', true),
+		}
+		if ($tn == NULL) {
+			// Handle old Shipping Tracking plugin 
+			$tn = get_post_meta($order->id, '_tracking_number', true);
+			if ($tn == NULL) {
```
Comment (Bossa573) [inline api/class-aftership-api-orders.php]: this part, can fix the indentation a little bit? kinda hard to read

Comment (Bossa573) [inline api/class-aftership-api-orders.php]: from line 272 to 282

Comment (bhanuc) [inline api/class-aftership-api-orders.php]: @Bossa573  done

**Pull request merged**